prv_accountant/__init__.py

```diff
@@ -2,4 +2,5 @@
 # Licensed under the MIT License.
 
 from .accountant import Accountant, PRVAccountant  # noqa: F401
-from .privacy_random_variables import PrivacyRandomVariable, PoissonSubsampledGaussianMechanism  # noqa: F401
+from .privacy_random_variables import PrivacyRandomVariable  # noqa: F401
```

Comment:
NIT could this be one line of imports?

Reply:
I agree but I couldn't find a way that makes the linter happy. I need the `noqa` otherwise it complains about an unused import. If I keep it on one line it exceeds the max line length and if I put it in `()` then the linter doesn't recognize the `noqa`